.buildkite/pipeline.yml

```diff
@@ -32,7 +33,7 @@ steps:
   - trigger: "kolibri-raspbian-image"
     label: ":raspberry-pi:"
     async: true
-    build: &triggered_build_attributes
+    build:
       message: "${BUILDKITE_MESSAGE}"
       branch: "autoblock"
```

Comment:
I think you mentioned in the rpi PR that you have plans to remove this line later. Do you plan to remove it in another PR? Thank you!

Reply:
`autoblock`? Yes, I'm keeping that branch alive until this gets merged into `develop`, since `develop` is still buildking off of the `master` branch in the rPi repo

Reply:
sounds good! thanks for the explanation!

Reply:
feel free to merge this to Develop